packages/core/contracts/financial-templates/contract-for-difference/ContractForDifference.sol

```diff
@@ -132,6 +139,7 @@ contract ContractForDifference is Testable, Lockable {
         collateralToken = _collateralAddress;
 
         financialProductLibrary = _financialProductLibraryAddress;
+        customAncillaryData = _customAncillaryData;
```

Comment:
you should probs bound the length of this. it would be possible to put an ancillaryData in here that the OO would reject. `ancillaryBytesLimit = 8192;` in the OO and this should be checked here as well or you could create a CFD that would deploy just fine but would reject when requesting a price.

Reply:
Added a check for this. PTAL. It's a little more complicated than just checking that hardcoded limit because the OO adds data too. So we need to check the length of the stamped OO data.